### ISSUE TITLE
'loss': 0.0 and same 'grad_norm" in task lora finetuning

### ISSUE THREAD

**Issue by Jeremy88888** [LLaVA-VL/LLaVA-NeXT#281]
I am training task lora on "liuhaotian/llava-v1.5-13b" by following the same code in LLaVA repo:
https://github.com/haotian-liu/LLaVA/blob/main/scripts/v1_5/finetune_task_lora.sh

The above runs fine in LLaVA repo (https://github.com/haotian-liu/LLaVA/tree/main). When I run it in this LLaVA-NeXT repo, it keeps showing 'loss': 0.0 and same 'grad_norm". Any idea?

![Screenshot_20241006_003928_Slack](https://github.com/user-attachments/assets/e3be8a2f-1f84-4a35-aa07-94b750fdaa42)

**Comment by refine-counting** [comment]
Hi, I got the same issue. Did you manage to resolve this?

**Comment by paperman-anne** [comment]
Hi, I got the same issue. What kind of GPU are you using?

**Comment by zyandtom** [comment]
have you solved yet?

**Comment by wuxi-dixi** [comment]
I got the same issue. have you solved yet?

**Comment by maocheno** [comment]
> I got the same issue. have you solved yet?

你好解决了吗？

**Comment by zyandtom** [comment]
probably max_length is too small, GT will be truncated?

**Comment by maocheno** [comment]
> probably max_length is too small, GT will be truncated?

my max_length is 512. But when i use lora , i find that appears
/root/lanyun-tmp/env/llava_next/lib/python3.10/site-packages/torch/utils/checkpoint.py:61: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn(
  0%|                                                                                                                                                                                                                                                        | 0/28479 [00:00<?, ?it/s]/root/lanyun-tmp/env/llava_next/lib/python3.10/site-packages/torch/utils/checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
  warnings.warn(
/root/lanyun-tmp/env/llava_next/lib/python3.10/site-packages/torch/utils/checkpoint.py:61: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn(
/root/lanyun-tmp/env/llava_next/lib/python3.10/site-packages/torch/utils/checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.

**Comment by maocheno** [comment]
> probably max_length is too small, GT will be truncated?

Bro, you are right. When I delete the, everyting is fine. That may be  will occupy too much tokens.

**Comment by maocheno** [comment]
> probably max_length is too small, GT will be truncated?

Bro, you are right. When I delete the<image>, everyting is fine. That may be  will occupy too much tokens.

**Comment by JiahuiKChen** [comment]
`--model_max_length 32768 \`
Needs to be set for any SigLIP run or else the loss is constant and grad_norm is 0.0
for any fine-tuning run, not just with LORA finetuning
